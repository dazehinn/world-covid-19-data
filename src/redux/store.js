import { configureStore } from '@reduxjs/toolkit';
import covidReducer, { fetchCovidData } from './statisticsSlice';

const store = configureStore({
  reducer: {
    Covid: covidReducer,
  },
});

store.dispatch(fetchCovidData());
export default store;
