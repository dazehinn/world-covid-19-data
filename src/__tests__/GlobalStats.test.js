import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store';
import GlobalStats from '../components/Globalstats';

describe('GloabalStats', () => {
it('Test Globalstats renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <GlobalStats />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
});