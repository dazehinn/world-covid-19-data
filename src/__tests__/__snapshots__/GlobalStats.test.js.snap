// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GloabalStats Test Globalstats renders correctly 1`] = `
<div
  className="loading"
>
  <div
    className="spinner-border"
  />
</div>
`;
